pkg/ccl/gssapiccl/get_user.go

```diff
@@ -54,7 +54,9 @@ func getGssUser(c pgwire.AuthConn) (connClose func(), gssUser string, _ error) {
 	//
 	// See https://github.com/postgres/postgres/blob/f4d59369d2ddf0ad7850112752ec42fd115825d4/src/backend/libpq/pqcomm.c#L269
 	connClose = func() {
-		C.gss_delete_sec_context(&lminS, &contextHandle, C.GSS_C_NO_BUFFER)
+		var minorStatus C.OM_uint32
+		C.gss_delete_sec_context(&minorStatus, &contextHandle, C.GSS_C_NO_BUFFER)
+		C.gss_release_name(&minorStatus, &srcName)
```

Comment:
My immediate reaction was like, "it's kinda a glaring mistake", then I saw this [1], and now I don't feel that bad. Never used that API but on a cursory look, it kinda looks dense.

[1] https://bugzilla.redhat.com/show_bug.cgi?id=911167